src/perftest_parameters.h

```diff
@@ -439,6 +439,7 @@ struct perftest_parameters {
 	int				disable_fcs;
 	int				flows;
 	int				flows_burst;
+        int                             wait_keystroke;
```

Comment:
need to Surround in line 392 the 
int    use_cuda  
with   
]#ifdef HAVE_CUDA